main_routers/jukebox_router.py

```
@@ -0,0 +1,1111 @@
+                        if current_default not in jukebox_config.data["actions"]:
+                            song["defaultAction"] = action_id
+                            logger.info(f"导入设置默认动画: {song_id} -> {action_id} (原默认动画不存在)")
+        
+        jukebox_config.save()
+        
+        logger.info(f"导入完成: {stats}")
+        return {"success": True, "stats": stats}
+
+
+@router.post("/pack-folder")
+async def pack_folder(files: List[UploadFile] = File(...)):
```

Ruff (0.15.9)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)